app/controllers/overrides/devise_token_auth_overrides/registrations_controller.rb

```diff
@@ -5,6 +5,12 @@ class RegistrationsController < DeviseTokenAuth::RegistrationsController
       before_action :permit_extra_params, only: %i[create update]
       skip_before_action :validate_account_update_params, only: :update
 
+      def resource_data(_opts = {})
+        response_data = UserSerializer.new(@resource).as_json['data']
```

Comment:
I wouldn't use `as_json['data']`. Just `UserSerializer.new(@resource).serializable_hash` or `.serializable_json` depending on what the function is expected to return.

Since the whole point was to stick to the jsonapi.org format, don't extract the data bit only. 
Check the rest of the api's responses to know what to expect it to look like 😄